cmd/otel-allocator/watcher/promOperator_test.go

```diff
@@ -724,6 +724,68 @@ func TestLoadConfig(t *testing.T) {
 				},
 			},
 		},
+		// {
```

Comment:
Is this supposed to be commented out?

Reply:
@swiatekm-sumo - Sorry, this is still WIP. I accidentally pushed it :)

Reply:
I have addressed the comments now, its ready to review.